frontend/Dockerfile

```
@@ -1,21 +1,21 @@
 ENV INFISICAL_TOKEN=$INFISICAL_TOKEN
 
 COPY package*.json ./
-RUN npm install
+RUN npm ci
 
 RUN apt-get update && apt-get install -y bash curl && curl -1sLf \
 'https://dl.cloudsmith.io/public/infisical/infisical-cli/setup.deb.sh' | bash \
 && apt-get update && apt-get install -y infisical
 
-COPY . .
+COPY --link --exclude=node_modules . . 
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)